scripts/installers/uninstall-mac-pkg

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+"""Script to uninstall AWS CLI V2 Mac PKG"""
+import argparse
+import sys
+import os
+from datetime import datetime
+from subprocess import check_output
+from subprocess import CalledProcessError
+from subprocess import PIPE
+
+SCRIPTS_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(SCRIPTS_DIR)
+
+from utils import run
+from utils import BadRCError
+
+
+_PKG_ID = 'com.amazon.aws.cli2'
+
+
+def uninstall():
+    assert _is_installed() is True, 'Could not find AWS CLI installation.'
+    assert os.geteuid() == 0, 'Script must be run as root (with sudo).'
+
+    root_dir = _get_root_dir()
+    list_files = _get_file_list(root_dir)
+    _erase_files(list_files)
+    list_dirs = _get_dir_list(root_dir)
+    _erase_dirs(list_dirs)
+    _forget()
+    return 0
+
+
+def _get_root_dir():
+    lines = run('pkgutil --pkg-info %s /' % _PKG_ID, echo=False).split('\n')
+    volume_line = lines[2]
```

Comment:
It would be nice to make this a regrex instead to pull out all of this information.